### DISCUSSION TITLE
Mysql 4.1

### DISCUSSION THREAD

**Discussion by sdutt0121** [mydumper/mydumper#1309] [Q&A]
Hello, I am in process of migrating 1TB of MySQL (4.1) database running on windows 2008 from on-prem to AWS rds on version 8.0.. 
as of now I am using mysqldump to export/import data .. which is really very time consuming.. and taking 20+ hours.. 
Can somebody please let me know what is the fastest way i can  migrate this database to AWS.

i was thinking about using mydumper but not sure if i can use that tool in my case

Please help.

**Comment by davidducos** [comment]
Hi @sdutt0121, 
Did you test MyDumper??
